src/components/BatchTx/BatchTotalAmount.tsx

```diff
@@ -0,0 +1,32 @@
+import { useMemo } from 'react';
+import { Typography } from 'antd';
+import styled from 'styled-components';
+
+import { useBatchParser } from './useBatchParser';
+import { useBatchConfig } from './useBatchConfig';
+
+function BatchTotalAmount() {
+  const { parsedTxs } = useBatchParser();
+  const { selectedAsset } = useBatchConfig();
+
+  const calcTotalAmount = useMemo(() => {
+    let totalAmount = 0;
+    for (let row of parsedTxs) {
+      let amount = Number(row.amount?.parsedValue!);
```

Comment:
cannot use `Number` with `BigNumber` instances. you need to use these methods instead: https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--methods

Reply:
`const`, not `let`